client/packages/common/src/ui/icons/BarChart2.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const BarChart2Icon = (props: SvgIconProps): JSX.Element => {
```

Comment:
Should it be called `TrendingUp` throughout?

Reply:
I changed the icon, due to those weird prop/styling issues I mentioned in the chat. This one is called BarChart2: https://feathericons.com/?query=barchart-2